test-mineflayer/storeys.test.js

```diff
@@ -13,46 +13,42 @@ const createBot = (command) => {
   return bot;
 }
 
-const handleError = (bot, done) => {
-  bot.on("error", (e) => {
-    done(e);
-  });
-}
-
 describe("Storeys plugin test", () => {
-  beforeAll(() => {
-    // Prevent jest from complaining about an import after the test is done (You are trying to `import` a file after the Jest environment has been torn down.)
-    jest.useFakeTimers('legacy');
-    console.log = () => { };
-  });
+  let bot;
+
+  const handleError = (done) => {
+    bot.on("error", (e) => {
+      done(e);
+    });
+  }
+  
+  afterAll(() => bot.quit());
```

Comment:
```suggestion
  afterEach(() => bot.quit());
```

makes this work for me locally!